scss/sports/customizations.scss

```diff
@@ -0,0 +1,295 @@
+        }
+    }
+  }
+
+  .bxi-button.bxi-button--icon.bxi-button--lg {
+    margin-top: 4px;
+    width: 45px;
+    height: 45px;
+  }
+
+  .bxi-button--link {
+    color: var(--bxi-primary-color);
+    font-size: inherit;
+    background: none;
+    padding: 0;
+      &:hover {
+        color: var(--bxi-secondary-color);
+        background: none;
+    }
+  }
+
+  bxi-user-icon {
+    svg {
+      @include color-icon(var(--bxi-primary-color))
+    }
+  }
+
+  .bxi-bg-white {
+     background-color: #fff;
+  }
+
+  .bxi-banner-title {
+    margin-bottom: 1.5rem;
+    font-family: var(--bxi-secondary-font);
+    font-size: 64px;
+    line-height: 76px;
+    font-weight: 700;
+    max-width: 600px;
+    -webkit-font-smoothing: antialiased;
+
+    @include breakpoint-lt-lg {
+      font-size: 32px;
+      line-height: 42px;
+    }
+  }
+
+  .bxi-banner-text {
+    margin-bottom: 1.5rem;
+    font-family: var(--bxi-primary-font);
+    font-size: 22px;
+    max-width: 560px;
+    -webkit-font-smoothing: antialiased;
+
+    &.lh-base {
+      line-height: 36px!important;
+
+      @include breakpoint-lt-lg {
+        line-height: 22px!important;
+      }
+    }
+
+    @include breakpoint-lt-lg {
+      font-size: 16px;
+    }
+  }
+
+  .bxi-banner-overlay {
+    background: linear-gradient(92.33deg, rgba(0, 3, 11, 0.8) 27.4%, rgba(0, 4, 14, 0) 82.41%);
+    min-height: 650px;
+  }
+
+  .footer-support {
+    bxi-link {
+      font-size: 12px;
+      font-weight: 400;
+    }
+
+    .bxi-button {     
+      width: 186px;
+      padding: 10px 24px;
+      border-radius: 30px;
+      border: 1px solid #fff;
+      background: transparent;
+      font-weight: 600;
+      font-size: 12px;
+      line-height: 24px;
+      color: #fff;
+      text-decoration: none;
+      text-align: center;
+      transition: 0.2s;
+      cursor: pointer;
+      
+      &:hover {
+        background: #fff;
+        color: var(--bxi-secondary-color);
+      }
+    }
+  }
+
+  .footer-contacts-text {
+    font-family: var(--bxi-secondary-font)!important;
+  }
+
+  .bxi-copyright-bg-color {
+    background-color: var(--bxi-sports-copyright-bg-color);
+  }
+
+  .bxi-tiles {
+    border-radius: 0.5rem;
+    display: flex;
+    align-items: flex-end;
+    width: 100%;
+    height: 250px;
+    padding-top: 50px;
+
+    &:hover {
+      padding-top: 0;
+    }
+
+    &-overlay {
+      background: linear-gradient(3.88deg, rgba(0, 3, 11, 0.8) 18.71%, rgba(0, 4, 14, 0) 69.21%);
+    }
+  }
+
+  .bxi-subscription {
+    border-radius: 0.5rem;
+    border: 1px solid #A6ACBA;
+    background: #fff;
+    display: flex;
+    align-items: flex-end;
+    width: 100%;
+    height: 300px;
+    padding-top: 20px;
+
+    .tile__title {
+      font-size: 1.17rem;
+    }
+
+    .tile__card {
+      color: #fff;
+      margin-bottom: 1.5rem;
+      border-radius: 12px;
+      padding: 1rem;
+      height: 125px;
+      width: 202px;
+      margin:auto;
+
```

Comment:
Can you add these as variables in the branding sheet in case they need to be rebranded?